engine-vuex/src/wwtaware.ts

```diff
@@ -582,6 +584,12 @@ export class WWTAwareComponent extends Vue {
    */
   zoom!: (f: number) => void;
 
+  /** Moves the position of the view */
```

Comment:
If possible it would be good to document the units of the `x` and `y` values here ... but they might not be well-defined. Barring that, some sort of documentation of "reasonable" values might be helpful for future users of these APIs.